src/cli.rs

```diff
@@ -259,8 +269,13 @@ impl RunCommand for MetricsCommand {
         info!("Running `metrics` subcommand");
         debug!("{:#?}", self);
 
+        let mut sp = Spinner::new(
+            DEFAULT_PROGRESS_SPINNER,
+            "Downloading metadata from Azure".into(),
```

Comment:
Perhaps reword this one too (since will not necessarily be from Azure)?
```suggestion
            "Downloading and searching metadata".into(),
```